readme.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const ruleMod = require('./rules');
```

Comment:
I wouldn't change the name, but I would be happy if we could do this in TypeScript. The folder could be `src/readme`.